test/test_ops.py

```diff
@@ -581,6 +585,7 @@ def _distance(a, b):
     # Tests that experimental Python References perform the same computation
     # as the operators they reference, when operator calls in the torch
     # namespace are remapped to the refs namespace (torch.foo becomes refs.foo).
+    @skipXPU
```

Comment:
Give some comments here to clarify why we skip xpu here in this file:
1. Emphasize only temporarily skip xpu in this file
2. Elaborate on the reason.